stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/CSS.java

```diff
@@ -33,18 +46,276 @@ public void disable(JsonRpcPeer peer, JSONObject params) {
   }
 
   @ChromeDevtoolsMethod
-  public JsonRpcResult getSupportedCSSProperties(JsonRpcPeer peer, JSONObject params) {
-    GetSupportedCSSPropertiesResponse response = new GetSupportedCSSPropertiesResponse();
-    response.cssProperties = Collections.emptyList();
-    return response;
+  public JsonRpcResult getComputedStyleForNode(JsonRpcPeer peer, JSONObject params) {
+    final GetComputedStyleForNodeRequest request = mObjectMapper.convertValue(
+        params,
+        GetComputedStyleForNodeRequest.class);
+
+    final GetComputedStyleForNodeResult result = new GetComputedStyleForNodeResult();
+    result.computedStyle = new ArrayList<>();
+
+    mDocument.postAndWait(new Runnable() {
+      @Override
+      public void run() {
+        mDocument.getElementStyles(
+            mDocument.getElementForNodeId(request.nodeId),
```

Comment:
we should call `getElementForNodeId` and check its return value before passing that along to `getElementStyles`